webrtc.html

```diff
@@ -4551,11 +4551,12 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
             </dd>
             <dt><code>removeTrack</code></dt>
             <dd>
-              <p>Stops sending media from <var>sender</var>. The
-              <code><a>RTCRtpSender</a></code> will still appear in
-              <code>getSenders</code>. Doing so will cause future calls to
-              <code>createOffer</code> to mark the <a>media description</a> for
-              the corresponding transceiver as <code>recvonly</code> or
+              <p>Stops sending media from <var>sender</var>, pending
+              negotiation. The <code><a>RTCRtpSender</a></code>
```

Comment:
I agree that `transceiver.stop`, `encodings[].active` and `setDirection` aren't solutions for this. But wouldn't disassociating the track from the sender work? If you call `addTransceiver`, you end up with a sender with a `null` track. So `removeTrack` could just return the sender to this state (of "can send, but have nothing to send"). Maybe this needs some list discussion. If we went this route it would also make sense to allow passing `null` into `replaceTrack`.